src/Dfe.PlanTech.Data.Sql/Repositories/SubmissionRepository.cs

```diff
@@ -94,15 +94,15 @@ public async Task ConfirmCheckAnswersAndUpdateRecommendationsAsync(int establish
             {
                 if (r.CompletingAnswers.Any(ca => responses.Contains(ca.Id)))
                 {
-                    return new { r.Id, Status = RecommendationConstants.CompletedKey };
+                    return new { r.Id, Status = RecommendationStatus.Complete.GetDisplayName() };
                 }
 
                 if (r.InProgressAnswers.Any(ca => responses.Contains(ca.Id)))
                 {
-                    return new { r.Id, Status = RecommendationConstants.InProgressKey };
+                    return new { r.Id, Status = RecommendationStatus.InProgress.GetDisplayName() };
                 }
 
-                return new { r.Id, Status = RecommendationConstants.NotStartedKey };
+                return new { r.Id, Status = RecommendationStatus.NotStarted.GetDisplayName() };
```

Comment:
The implementation seems to have changed - we are now storing the display name (which contains a non-breaking space) instead of a stable "key":

<img height="400" alt="Image" src="https://github.com/user-attachments/assets/41fcc1d8-7328-4c08-b38d-f360636b18cc" />

This, therefore, tightly couples what the user sees in UI with what is stored in the database. 

I've not tested this specifically (so suitably sized pinch of salt needed!) but I fear this will, therefore, force overheads on checks such as doing case-insensitive matching/searching.

Reply:
Using the enum as the key - resolved

<img width="593" height="520" alt="image" src="https://github.com/user-attachments/assets/f6503b71-1bd3-4139-9b94-bfef918d579c" />